spec.html

```diff
@@ -49229,53 +49229,36 @@ <h1>
           </dl>
           <emu-alg>
             1. Let _values_ be a new empty List.
-            1. Let _remainingElementsCount_ be the Record { [[Value]]: 1 }.
+            1. Let _remainingElementsCount_ be 0.
```

Comment:
Does the introduce a race? Similar to https://github.com/tc39/proposal-await-dictionary/pull/24#discussion_r2466262713

If we don't have a counter that represents that we haven't got to iterator `~done~` yet then a badly behaving promise that calls it's `.then` callback sync would set the count back to `0` before we've finished iterating.

When currently the spec waits until it gets to the end.